src/diffusers/models/model_loading_utils.py

```diff
@@ -252,6 +254,10 @@ def load_model_dict_into_meta(
                 param = param.to(dtype)
                 set_module_kwargs["dtype"] = dtype
 
+        if is_accelerate_version(">=", "1.9.0.dev0"):
```

Comment:
```suggestion
        if is_accelerate_version(">", "1.8.0"):
```
This way we don't need to put dev. I'll also do a accelerate release next week !